bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -83,14 +78,13 @@ else if (!isRegistered(userService.getCurrentUser().getUserId()) && !request.get
     }
 }
```

Comment:
Looks like you need to run a formatter on this whole block, spacing wise.

Reply:
ran formatter